src/indexer/index.ts

```diff
@@ -90,6 +92,11 @@ async function init(): Promise<{ domainIndexers: Array<DomainIndexer>; app: Fast
 
   const domainsToIndex = getDomainsToIndex(sharedConfig.domains)
   const domainIndexers: Array<DomainIndexer> = []
+
+  const notificationSender = new NotificationSender(process.env.SNS_REGION!)
+
+  startCronJob("* */10 * * * *", checkTransferStatus, transferRepository, notificationSender)
```

Comment:
Let us also enable this string `"* */10 * * * *"` that defines cron job frequency to be configurable. We can use another ENV variable just have a default value of 10 min.